File: app/src/main/java/com/thalesgroup/gemalto/fido2/sample/Configuration.java
package com.thalesgroup.gemalto.fido2.sample;

public class Configuration {

    // Refer the release documentation, how to generate the public key and place it below for publicKeyModulus & publicKeyExponent
    public static final byte[] publicKeyModulus = new byte[] {

    };

    public static final byte[] publicKeyExponent = new byte[] {

    };

    //SafetyNetAttestationKey. Get new key from here: https://developer.android.com/training/safetynet/attestation#obtain-api-key, and put below.
    public final static String safetyNetAttestationKey = "";

    //Place your relying party address here
    public final static String rpId = "";
}

File: app/src/main/java/com/thalesgroup/gemalto/fido2/sample/Configuration.kt
/*
 * Copyright © 2021 THALES. All rights reserved.
 */
package com.thalesgroup.gemalto.fido2.sample

object Configuration {
    @JvmField
    //Replace this byte array with your own public key modulus.
    val publicKeyModulus: ByteArray = byteArrayOf(

    )

    @JvmField
    //Replace this byte array with your own public key exponent.
    val publicKeyExponent: ByteArray = byteArrayOf(

    )

    // Fido2 rpId, Replace this value to your own rpId
    const val rpId: String = "genuflecto.github.io"
}
